### DISCUSSION TITLE
ASC3 and EOS Data transfer

### DISCUSSION THREAD

**Discussion by AndrewMC6** [udotdevelopment/ATSPM#216] [Q&A]
I have ATSPM fully installed and running currently, I have added a couple lab ASC3 controllers as proof of concept of data being imported into the database, and to be sure everything is working... running into a few snags, I had noticed the controller events table was empty, prompting me to install wireshark on VM and monitor the transfer. I can see the FTP connection be established, authenticated, some commands go back and forth, the controller gives an error at an EPRT command, then I see it can't build a data connection, then after some time it seems to timeout and close the connection.

I don't see scripts to gather data from EOS controllers? I have one controller set to EOS but don't see any wireshark activity of attempts to download its data.

Any insight greatly appreciated!

**Comment by avetraffic** [comment]
We've seen issues with newer versions of EOS controllers not downloading data properly. As of right now 32.67.20 Is the latest version that I believe is still working correctly.
